pep-0684.rst

```diff
@@ -0,0 +1,640 @@
+may crash or, worse, experience inconsistent behavior.  That was part
+of the motivation for :pep:`630` and friends.  So this is not a new
+situation nor a consequence of this proposal.
+
+In contrast, when the `proposed API <proposed capi_>`_ is used to
+create multiple interpreters, the default behavior will change for
+some extensions.  In that case, importing an extension will fail
+(outside the main interpreter) if it doesn't indicate support for
+multiple interpreters.  For extensions that already break in
+multiple interpreters, this will be an improvement.
+
+Now we get to the break in compatibility mentioned above.  Some
+extensions are safe under multiple interpreters, even though they
+haven't indicated that.  Unfortunately, there is no reliable way for
+the import system to infer that such an extension is safe.  So
+importing them will still fail.  This case is addressed in
+`Extension Module Compatibility`_ below.
+
+Extension Module Maintainers
+----------------------------
+
+One related consideration is that a per-interpreter GIL will likely
+drive increased use of multiple interpreters, particularly if :pep:`554`
+is accepted.  Some maintainers of large extension modules have expressed
+concern about the increased burden they anticipate due to increased
+use of multiple interpreters.
+
+Specifically, enabling support for multiple interpreters will require
+substantial work for some extension modules.  To add that support,
+the maintainer(s) of such a module (often volunteers) would have to
+set aside their normal priorities and interests to focus on
+compatibility (see :pep:`630`).
+
+Certainly extension maintainers are free to not add support for use
+in multiple interpreters.  However, users will increasingly demand
+support for multiple interpreters, especially if the feature grows
+in popularity.
+
+Either way the situation can be stressful for maintainers of such
+extensions, particularly when they are doing the work in their spare
+time.  The concerns they have expressed are understandable.  We address
+the partial solution in `Restricting Extension Modules`_ below.
+
+Alternate Python Implementations
+--------------------------------
+
+Other Python implementation are not required to provide support for
+multiple interpreters in the same process (though some do already).
+
+Security Implications
+---------------------
+
+There is no known impact to security with this proposal.
+
+Maintainability
+---------------
+
+On the one hand, this proposal has already motivated a number of
+improvements that make CPython *more* maintainable.  That is expected
+to continue.  On the other hand, the underlying work has already
+exposed various pre-existing defects in the runtime that have had
+to be fixed.  That is also expected to continue as multiple interpreters
+receive more use.  Otherwise there shouldn't be a significant impact
+on maintainability.  The effect should be net positive.
```

Comment:
```suggestion
receive more use.  Otherwise, there shouldn't be a significant impact
on maintainability, so the net effect should be positive.
```

Add missing comma, avoid somewhat confusing/contradictory phrasing exacerbated by choppy flow.